src/addOnMessageListenerWithWebViewKey.ts

```diff
@@ -0,0 +1,17 @@
+import { NativeModules, Platform, EmitterSubscription, NativeEventEmitter } from "react-native";
+
+const scriptMessageEmitter = new NativeEventEmitter(
+  Platform.select({
+    ios: NativeModules.ScriptMessageEventEmitter,
+    default: null
+  })
+);
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export default function addOnMessageListenerWithWebViewKey(webViewKey: string, listener: (event: any) => void): EmitterSubscription {
```

Comment:
Ah, this can be updated

Reply:
I think we may need a custom type because we also add the `webViewKey` to the event. https://github.com/discord/react-native-webview/pull/6/files#diff-c1064c73d2b45b9e3b6352aa2f27995b05ec16f5e1fd958dd2fa86070b842aa7R22